corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -327,6 +327,126 @@ hqDefine("geospatial/js/case_grouping_map",[
         );
     }
 
+    function caseGroupSelectModel() {
+        'use strict';
+        var self = {};
+
+        self.allGroups = ko.observableArray([]);
+        self.allCaseGroups;
+        self.visibleGroupIDs = ko.observableArray([]);
+        self.groupMaxSizeBreached = ko.observable(false);
+        self.casePerGroup = {};
+
+        self.groupIDInVisibleGroupIds = function(groupID) {
+            if (self.visibleGroupIDs().indexOf(groupID) !== -1) {
+                return true;
+              } else {
+                return false;
+              }
+        };
+
+        self.getGroupByID = function(groupID) {
+            return _.filter(self.allGroups(), function(group) {return group.groupID === groupID})[0];
+        };
+
+        self.loadCaseGroups = function(caseGroups) {
+            self.allCaseGroups = caseGroups; // {case_id: {group_info}}
+            // Add groups to the cases being exported
+
+            var groupIds = [];
```

Comment:
Nit: Use let here instead to keep it consistent with the other variable declarations in this function.

Reply:
https://github.com/dimagi/commcare-hq/pull/33680/commits/b514f9e1390c71bc57c0e914b872458f0532afa6